src/microbe_stage/Membrane.cs

```diff
@@ -82,17 +83,6 @@ public bool Dirty
     /// </remarks>
     public List<Vector2> OrganellePositions { get; set; } = PreviewMembraneOrganellePositions;
```

Comment:
Should this variable be renamed now? Or is this clear enough still. I guess technically each hex position is still an "organelle position". I guess it is fine if no one wants to rename this.